shopfloor_mobile/static/wms/src/scenario/delivery.js

```diff
@@ -29,6 +29,14 @@ const Delivery = {
                     :key="make_state_component_key(['picking-summary', 'detail-picking', state.data.picking.id])"
                     />
             </div>
+            <div v-if="state_is('deliver') && display_location_operations_progress()">
+                <item-detail-card
```

Comment:
can you add and option to `item-detail-card` to display fields inline? I think it's better for such cases as you save some screen space.

Reply:
In the beginning it was a one-liner, then it was decided we would keep it in two separate lines for clarity. What do you think?

Reply:
As long as they are readable (eg: proper separation w/ 2 columns of half width) I don't get why you would like burn more space...

Reply:
In most cases, ongoing will always be 0. I would display it only if != 0

Reply:
That's already the case (by default in this component, fields whose value is 0 are not displayed).
![image](https://github.com/OCA/wms/assets/77412816/24a6d720-68af-4334-ab0d-970c477349b6)